docs/_source/guides/llms/practical_guides/create_dataset.md

```diff
@@ -157,13 +158,227 @@ As an example, here is how you can transform a whole dataset into records at onc
 records = [rg.FeedbackRecord(fields={"question": record["instruction"], "answer": record["response"]}) for record in hf_dataset if record["category"]=="open_qa"]
 ```
 
+
 Now, we simply add our records to the dataset we configured [above](#configure-the-dataset):
 
 ```python
 #add records to the dataset
 dataset.add_records(records)
 ```
 
+## Add suggestions
+
+Suggestions refer to suggested responses (e.g. model predictions) that you can add to your records to make the annotation process faster. These can be added during the creation of the record or at a later stage. The suggestion should follow the schema of the response of the type of question answered by the suggestion:
+
+::::{tab-set}
+
+:::{tab-item} Label
+
+```python
+record = rg.FeedbackRecord(
+    fields=...,
+    suggestions = [
+        {
+            "question_name": "relevant",
+            "value": "YES",
+        }
+    ]
+)
+```
+:::
+
+:::{tab-item} Multi-label
+
+```python
+record = rg.FeedbackRecord(
+    fields=...,
+    suggestions = [
+        {
+            "question_name": "content_class",
+            "value": ["hate", "violent"]
+        }
+    ]
+)
+```
+
+:::
+
+:::{tab-item} Ranking
+
+```python
+record = rg.FeedbackRecord(
+    fields=...,
+    suggestions = [
+        {
+            "question_name": "preference",
+            "value":[
+                {"rank": 1, "value": "reply-2"},
+                {"rank": 2, "value": "reply-1"},
+                {"rank": 3, "value": "reply-3"},
+            ],
+        }
+    ]
+)
+```
+
+:::
+
+:::{tab-item} Rating
+
+```python
+record = rg.FeedbackRecord(
+    fields=...,
+    suggestions = [
+        {
+            "question_name": "quality",
+            "value": 5,
+        }
+    ]
+)
+```
+
+:::
+
+:::{tab-item} Text
+
+```python
+record = rg.FeedbackRecord(
+    fields=...,
+    suggestions = [
+        {
+            "question_name": "corrected-text",
+            "value": "This is a *suggestion*.",
+        }
+    ]
+)
+```
+
+:::
+
+::::
+
+You can also add suggestions to existing records that have been already pushed to Argilla:
+
+```python
+import argilla as rg
+
+rg.init()
+
+records = fetch_records(name="my_dataset", workspace="my_workspace")
+# if you need to get the questions and fields as well you can use this line instead:
+# dataset = rg.FeedbackDataset.from_argilla(name="my_dataset", workspace="my_workspace")
+# and in the following lines subsitute `records` for `dataset`
+
+records = [record['suggestions']: [{"question_name": "question", "value": ...}] for record in records]
+
+records.push_to_argilla()
```

Comment:
So `fetch_records` is a method from `FeedbackDataset` class, and cannot be used separately, this can either be replaced by the code snippet below, or just assuming `ds` is an already initialized `FeedbackDataset` that is being pushed to Argilla, and right after that `suggestions` are included (`FeedbackDataset(...)` -> `add_records` -> `push_to_argilla` -> `fetch_records` -> `record.set_suggestions`).

Anyway, the following is better and clearer:

```suggestion
dataset = rg.FeedbackDataset.from_argilla(name="my_dataset", workspace="my_workspace")

for record in dataset.records:
    record.set_suggestions([{"question_name": "question", "value": ...}])

dataset.push_to_argilla() # No need to provide `name` and `workspace` as has been retrieved via `from_argilla` classmethod
```